misc/docker-compose/docker-compose.yml

```diff
@@ -1,25 +1,47 @@
-version: "3.7"
+version: '3.2'
 services:
-  gnonode:
-    container_name: gnoland-node
+  node:
+    ports:
+      - target: 26657
+        published: "26657"
+        mode: host
+    command: start -genesis-remote 0.0.0.0:26657 -root-dir .
     build:
```

Comment:
might be good to add `image: ghcr.io/gnolang/gno/gnoland-slim:latest`, wdyt?

Reply:
I also would put `build` and `image` at the top of the service, and only put `ports/command`, although that is a matter of preference.

Reply:
If you do that, we will be using the precompiled latest image from the hub repository. Using this approach we are building from local sources, allowing new changes to be tested too.

Reply:
Yes, I suggest you to keep the build directive rather than switching to the precompiled image.

But definitely prefer moving those statements at the top.